### PR TITLE
Docs: Plugin-tools development material - chunk 8

### PR DESCRIPTION
**Pull request by josmperez**
### What is this feature?
This is a PR holding a review of several plugins docs, focusing on structure, editorial soundness, and conformity to doc team style guidelines.

### Why do we need this feature?
This fulfills the doc improvement initiative described in the plugins doc plan.

### Who is this feature for?
Plugin developers

### CODE REVIEW

File: docusaurus/docs/nested-plugins.mdx [hunk 1]
```diff
@@ -3,7 +3,11 @@ id: nested-plugins
 title: Nested Plugins
 ---
 
-Grafana app plugins can nest frontend datasource and panel plugins making it easy to ship a complete user experience. To take advantage of this feature first scaffold an app plugin using `@grafana/create-plugin`:
+Grafana app plugins can nest frontend data sources together with panel plugins so that you can provide a complete user experience.
```
Comment (academo) [inline docusaurus/docs/nested-plugins.mdx:6]: `can nest frontend data sources` also backend ones. The original text is incorrect here. Can you remove the frontend part from this sentence please? `can nest data sources`

File: docusaurus/docs/backend.md [hunk 2]
```diff
@@ -3,23 +3,31 @@ id: backend
 title: Backend
 ---
 
-The backend part of a Grafana plugin is written in Go. To keep the [Grafana plugin SDK for Go](https://grafana.com/docs/grafana/latest/developers/plugins/backend/grafana-plugin-sdk-for-go/) up to date:
+The preferred language for developing backend components of a Grafana plugin is [Go](https://go.dev/).
```
[the remark this answers is not in the document]
Comment (josmperez) [reply docusaurus/docs/backend.md:6]: fixed

File: docusaurus/docs/ci.md [hunk 3]
```diff
@@ -3,32 +3,34 @@ id: ci
 title: Continuous Integration
 ---
 
-@grafana/create-plugin comes with several GitHub workflows that help automate the plugin development process.
+Automate your development process to minimize errors and make it faster and more cost-efficient. Use the Continuous Integration (CI) features of the Grafana `create-plugin` tool to help automate your development process.
```
[the remark this answers is not in the document]
Comment (josmperez) [reply docusaurus/docs/ci.md:6]: fixed

File: docusaurus/docs/frontend.mdx [hunk 4]
```diff
@@ -3,30 +3,32 @@ id: frontend
 title: Frontend
 ---
 
-The frontend part of a Grafana plugin is written in Typescript with React as the view library. Within the plugin directory, you can run:
+Develop the frontend component of a Grafana plugin in Typescript with ReactJS as the View library. Several commands are available within the `plugin` directory for interacting with the frontend.
```
[the remark this answers is not in the document]
Comment (josmperez) [reply docusaurus/docs/frontend.mdx:6]: added

File: docusaurus/docs/nested-plugins.mdx (hunk 1 above)
Comment (josmperez) [reply docusaurus/docs/nested-plugins.mdx:6]: removed

**Pull request merged**